garrysmod/gamemodes/terrortown/gamemode/shared.lua

```diff
@@ -218,3 +210,9 @@ DefaultEquipment = {
       "weapon_ttt_glock"
    }
 };
+
+-- Mute types
+MUTE_NONE   = 0
+MUTE_TERROR = TEAM_TERROR
+MUTE_ALL    = 2
+MUTE_SPEC   = TEAM_SPEC

```

Comment:
I feel like this is error prone. It might be less error prone to have `ALL` equal the bitwise OR of TEAM_TERROR and TEAM_SPEC but numbers are probably better and make more sense simply by the fact that enums shouldn't depend on each other.